src/test/ui/issues/issue-56202.stderr

```diff
@@ -0,0 +1,7 @@
+error[E0601]: `main` function not found in crate `issue_56202`
+   |
+   = note: consider adding a `main` function to `$DIR/issue-56202.rs`
```

Comment:
Would it be better to make this a `// compile-pass`, either as a library or with an empty `main`?

Reply:
Seems better to me yeah; @alexreg can you change this in your upcoming PR?

Reply:
@alexreg 👍